### DISCUSSION TITLE
Windows application crashing on CEF initialization due to "CefSettings.BrowserSubprocessPath not found." error

### DISCUSSION THREAD

**Discussion by susmitakanchinadham** [cefsharp/CefSharp#4979] [Q&A]
We are seeing a few of our customers facing "CefSettings.BrowserSubprocessPath not found." error while initializing CEF in our application.
From online posts on this error, AV quarantining this file could be a possible reason for this issue.
Would anyone know if there is a fix or workaround for this issue.
One more general question is, how can we get digitally signed binaries of CefSharp to avoid such AV issues.

**Comment by susmitakanchinadham** [comment]
@amaitland Would you have any insight on this?

**Comment by amaitland** [comment]
Use your own application exe as the `Subprocess`. 

https://github.com/cefsharp/CefSharp/wiki/SelfHost-BrowserSubProcess
